### DISCUSSION TITLE
Slack integration

### DISCUSSION THREAD

**Discussion by kennysden** [Security-Onion-Solutions/securityonion#2307] [Unsupported Versions] [answered]
![image](https://user-images.githubusercontent.com/59517722/101827167-f49d0480-3afd-11eb-9700-dedf0bb282c0.png)

I'm having issue setting up my webhook for slack on our Security Onion 2.3.10. This is the error I got once I ran the so-elastalert-test. Has anyone tested this feature?

**Comment by kennysden** [comment, marked as answer]
I have put that webhook inside the YAML file

**Reply by jertel** [reply]
Please see #2032. The script will function correctly in the next release.

**Reply by kennysden** [reply]
Okay, I will have to wait, I tried the above solution and got the below screenshot. 
![image](https://user-images.githubusercontent.com/59517722/101850245-650c4b80-3b27-11eb-847e-cbd882872aab.png)
